Assets/Code/Scripts/PlayerManagement/MouseLook.cs

```diff
@@ -52,12 +67,26 @@ void Update()
             _yRotation += mouseX;
             _xRotation -= mouseY;
             _xRotation = Mathf.Clamp(_xRotation, -90f, 90f);    // Clamping allows to block the rotation
-        
-            // rotate camera and orientation
-            /*This order is needed because if the object is the same, the second assign value overwrites the values of the first one.
-            If the object is different, the order is irrelevant*/
-            _orientation.rotation = Quaternion.Euler(0, _yRotation, 0);
-            _transform.rotation = Quaternion.Euler(_xRotation, _yRotation, 0);
+            
+            // Checks whether there's local information about vertical axis preference.
+            if (PlayerPrefs.HasKey("masterInvertY"))
+            {
+                // The following rotation order is needed because if the object is the same,
+                // the second assign value overwrites the values of the first one. If the object is different, the order is irrelevant.
+                
+                if (PlayerPrefs.GetInt("masterInvertY") == 1)
```

Comment:
```suggestion
                if (PlayerPrefs.GetInt("masterInvertY"))
```

Reply:
I didn't know you could do that, I'm committing the suggestion.